src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -638,6 +638,10 @@ const MetaTraderUI = (() => {
         }
     };
 
+    const showEA = (is_shown) => {
```

Comment:
I prefer `should_show`, what do you think?

Reply:
tbh, this thing is just a one liner, do we really need this? I think should_show rolls of the tongue better.

Reply:
yes, better keep the function since that's the behaviour in metatrader to hide/show elements in .ui